cypress/e2e/buy-shirt.cy.ts

```diff
@@ -10,33 +10,30 @@ const generalStepPage = new GeneralStepPage();
 
 describe("Buy a t-shirt", () => {
   it("Then the t-shirt should be bought", () => {
```

Comment:
try to follow the same approach that the `cypress/e2e/dresses-list.cy.ts` file has to initialize the page objects using a before hook

Reply:
Done

I have a question
why can't I initialize the variables at the same time in the before?